inventory_management_system_api/routers/v1/usage_status.py

```diff
@@ -73,3 +78,26 @@ def get_usage_statuses(
 
     usage_statuses = usage_status_service.list()
     return [UsageStatusSchema(**usage_status.model_dump()) for usage_status in usage_statuses]
+
+
+@router.delete(
+    path="/{usage_status_id}",
+    summary="Delete a usage status by its ID",
+    response_description="Usage status deleted successfully",
+    status_code=status.HTTP_204_NO_CONTENT,
+)
+def delete_usage_status(
+    usage_status_id: str, usage_status_service: Annotated[UsageStatusService, Depends(UsageStatusService)]
+) -> None:
```

Comment:
```suggestion
def delete_usage_status(
    usage_status_id: Annotated[str, Path(description="ID of the usage status to delete")],
    usage_status_service: UsageStatusService = Depends()
) -> None:
```

Reply:
I would still use `usage_status_service: Annotated[UsageStatusService, Depends(UsageStatusService)]` as its preferred in the docs #117

Reply:
Sorry, that was a copy and paste error. Good spot, thanks!